### PR TITLE
LMM planner and Updated OCR

### PR DESCRIPTION
**Pull request by dillonalaird**

### CODE REVIEW

File: vision_agent/agent/vision_agent.py [hunk 1]
```diff
@@ -76,12 +78,35 @@ def extract_json(json_str: str) -> Dict[str, Any]:
...
+    media: Optional[Sequence[Union[str, Path]]]
+) -> Optional[Sequence[Union[str, Path]]]:
+    if media is None:
+        return None
+
+    new_media = []
+    for m in media:
+        m = Path(m)
+        extension = m.suffix
+        if extension in [".jpg", ".jpeg", ".png"]:
```
Comment (shankar-vision-eng) [inline vision_agent/agent/vision_agent.py:91]: I would add *.bmp here too as some verticals stick to bmp a lot

File: vision_agent/tools/tools.py [hunk 2]
```diff
@@ -198,14 +198,17 @@ def extract_frames(
 
 def ocr(image: np.ndarray) -> List[Dict[str, Any]]:
     """'ocr' extracts text from an image. It returns a list of detected text, bounding
-    boxes, and confidence scores. The results are sorted from top-left to bottom right
+    boxes with normalized coordinates, and confidence scores. The results are sorted
+    from top-left to bottom right. Each element in the returned list represents a
+    contiguous text region. Text on separate lines is considered separate regions and
```
Comment (shankar-vision-eng) [inline vision_agent/tools/tools.py:203]: `"Text on separate lines or text with more than a tab spacing between words is considered separate regions"`

**Review by shankar-vision-eng — CHANGES_REQUESTED** [review]
LGTM. Couple of minor things. I will also test this branch on some examples to make sure there is no regression

File: vision_agent/tools/tools.py (hunk 2 above)
Comment (dillonalaird) [reply vision_agent/tools/tools.py]: Reverted this change

**Review by shankar-vision-eng — APPROVED** [review]
LGTM

**Pull request merged**